src/Yaml.ts

```diff
@@ -244,7 +260,7 @@ export function formatYamlKey(key: string, value: any, options = { keepArrayInde
         throw new Error(`failed to create YAML for {${key}: ${value}}: ${(e as Error).message}`);
     }
 
-    y = arrayIndent(y, options.keepArrayIndent);
+    y = arrayIndent(y, options.keepArrayIndent == null ? false : options.keepArrayIndent);
```

Comment:
Is adding this comparison and ternary operator necessary? Why not just leave as is?